storage/registers.go

```diff
@@ -0,0 +1,43 @@
+package storage
+
+import (
+	"github.com/onflow/flow-go/model/flow"
+)
+
+// RegisterIndex defines methods for the register index.
+type RegisterIndex interface {
```

Comment:
there is some conflicts with @koko1123 's PR
https://github.com/onflow/flow-go/pull/4671/files#diff-921bdc69d712e22e7fe6c2ee7c2761c4c68e9c2f39ba73a2031eb79361c8f330R8-R11

Reply:
I'll change that on my end!